### PR TITLE
Core 1410 Apply Bulk Metadata.

### PR DESCRIPTION
**Pull request by sri2k1us**
image

PS. This is on top #349.

### CODE REVIEW

**Comment by sri2k1us** [comment]
I think we should add some kind of help about using / downloading templates to create the metadata file in this dialog. Feedback welcome.

File: src/components/data/toolbar/DataDotMenu.js [hunk 1]
```diff
@@ -397,6 +427,14 @@ function DataDotMenu(props) {
                 }}
             />
 
+            <ApplyBulkMetadataDialog
+                open={bulkMdDialogOpen}
+                handleClose={onBulkMdDialogClose}
+                destFolder={
+                    selectedResources[0] ? selectedResources[0]?.path : ""
```
Comment (psarando) [inline src/components/data/toolbar/DataDotMenu.js:434]:
```suggestion
                    selectedResources[0]?.path || ""
```

File: src/components/metadata/ApplyBulkMetadataDialog.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                    variant: AnnouncerConstants.INFO,
+                });
+                handleClose();
+            },
+            onError: setError,
+        }
+    );
+
+    const handleSubmit = (values) => {
+        console.log(JSON.stringify(values));
```
Comment (psarando) [inline src/components/metadata/ApplyBulkMetadataDialog.js:50]: Debug logging?
```suggestion
```

**Review by psarando — APPROVED** [review]
LGTM 👍

**Review by ash3rz — APPROVED** [review]
Currently has a conflict, but LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**